MiniApp/Classes/core/JavascriptBridge/MiniAppScriptMessageHandler.swift

```diff
@@ -82,6 +82,8 @@ internal class MiniAppScriptMessageHandler: NSObject, WKScriptMessageHandler {
             sendMessageToContactId(with: callbackId, parameters: requestParam)
         case .sendMessageToMultipleContacts:
             sendMessageToMultipleContacts(with: callbackId, parameters: requestParam)
+        case .getPoints:
+            fetchPoints(callbackId: callbackId)
```

Comment:
here is the suggestion if you want to change the method signature
```suggestion
            fetchPoints(with: callbackId)
```